src/Model/Request/ReferenceFile.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEuropa\CdtClient\Model\Request;
+
+/**
+ * Class ReferenceFile.
+ *
+ * Represents the reference file sent to the CDT API.
+ */
+class ReferenceFile
+{
+    /**
+     * The file.
+     */
+    protected File $file;
+
+    /**
+     * The reference languages.
+     *
+     * @var string[]
+     */
+    protected array $referenceLanguages;
+
+    public function getFile(): File
+    {
+        return $this->file;
+    }
+
+    public function setFile(File $file): ReferenceFile
+    {
+        $this->file = $file;
+        return $this;
+    }
+
+    /**
+     * @return string[]
```

Comment:
comments like this are perfect, it adds more clarification to something that is not clear from code